SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/RegisterAutoDestroyActorsTest/RegisterAutoDestroyActorsTest.cpp

```diff
@@ -53,14 +81,18 @@ void ARegisterAutoDestroyActorsTestPart1::BeginPlay()
 		}, 5.0f);
 	}
 
-	{ // Step 3 - Destroy by second server that doesn't have authority
-		AddServerStep(TEXT("SERVER_2_RegisterAutoDestroyActors"), 2, [](ASpatialFunctionalTest* NetTest) -> bool {
+	{ // Step 4 - Destroy by all servers that have authority
+		AddServerStep(TEXT("SERVER_ALL_RegisterAutoDestroyActors"), 0, [](ASpatialFunctionalTest* NetTest) -> bool {
```

Comment:
For future reference, always use FWorkerDefinition::ALL_WORKER_ID instead of directly setting 0.

Reply:
As we are spawning two actors on server 1 and then 1 actor will move to server 2 we should test by destroying actors on all servers. Updated to the 0 to FWorkerDefinition::ALL_WORKER_ID.